src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php

```diff
@@ -189,7 +189,7 @@ public function postAction(Request $request)
      * Change a filter by the given id.
      *
      * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param int $id the attribute id
+     * @param int                                       $id      the attribute id
```

Comment:
@GrahamCampbell Ok, then I will adjust that :)
@dantleech Yeah, but that's a different topic. Or can PHPCSFixer or StyleCI also do that?